spec/validate-spec.coffee

```diff
@@ -0,0 +1,31 @@
+describe 'validate', ->
+  validate = require('../lib/validate')
+  describe '::linter', ->
```

Comment:
whitespace is your necessary for readablilty. separate describes and tests with empty lines. separate setup from expectation with whitespace.